app/src/main/java/com/aceinteract/android/stepper/presentation/samples/TabStepperWithoutNavigationComponentsActivity.kt

```diff
@@ -0,0 +1,97 @@
+package com.aceinteract.android.stepper.presentation.samples
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.isVisible
+import com.aceinteract.android.stepper.R
+import com.aceinteract.android.stepper.StepperNavListener
+import kotlinx.android.synthetic.main.tab_stepper_without_navigation_components_activity.*
```

Comment:
Do you want to consider using viewbinding instead of kotlin synthetic here? Just for uniformity with the other activities.

Reply:
I can rewrite on viewBinding, but all other activities in samples package use kotlin synthetic.

Reply:
Oh haha. My bad. I was thinking of something else. Yeah, this is fine